frontend/src/components/viewers/Viewer.ts

```diff
@@ -23,6 +23,7 @@ export enum PlotType {
   TABLE = 'table',
   TENSORBOARD = 'tensorboard',
   WEB_APP = 'web-app',
+  VISUALIZATION_CREATOR = 'visualization-creator',
```

Comment:
itty bitty nit: move this above WEB_APP for alphabetization, same in ViewerContainer